src/frontend/components/data-table/lib/component.js

```diff
@@ -278,11 +276,39 @@ class DataTableComponent extends Component {
 
     this.toggleFilter(column)
 
+    let setField = title.trim();
+    if (setField.endsWith('Sort')) {
+      setField = setField.substring(0, setField.length - 4);
+    }
```

Comment:
See previous comments about column ID. It should be possible to get this with something like `.oAjaxData.columns[index].name` (see examples in `src/frontend/components/data-table/lib/component.js`)